nss-day-cohort-39/nutshell-razzmatazz-roadrunners
# Added News entry function

Changes:
- Added news article section to store saved articles
- Added dialog box functionality to save article input details
- Added a save button to save articles to json server

Test:
- Ensure all buttons still work
- Ensure dialog popup functions and the fields are fillable
- Ensure save button saves submission to local json api
- Ensure saved items are automatically rendered to the DOM

Example of what's needed in database.json:
"news": [
    {
      "title": "",
      "url": "",
      "comment": "",
      "timestamp": ,
      "id": 
    },
